features/guidance/turn-lanes.feature

```diff
@@ -350,10 +358,12 @@ Feature: Turn Lane Guidance
             | bg    | right |                     | no     |
 
         When I route I should get
-            | waypoints | route            | turns                    | lanes                            |
-            | a,f       | road,turn,turn   | depart,turn left,arrive  | ,left:true straight;right:false, |
-            | a,g       | road,right,right | depart,turn right,arrive | ,left:false straight;right:true, |
+            | waypoints | route            | turns                           | lanes                            |
+            | a,f       | road,turn,turn   | depart,turn left,arrive         | ,left:true straight;right:false, |
+            | a,d       | road,road,road   | depart,use lane straight,arrive | ,left:false straight;right:true, |
+            | a,g       | road,right,right | depart,turn right,arrive        | ,left:false straight;right:true, |
```

Comment:
I don't understand the `straight` here. I expected `straight: true/false` to be honest. Did something change in the Cucumber parsing logic?